dissect/target/plugins/general/network.py

```diff
@@ -48,18 +51,22 @@ def interfaces(self) -> Iterator[InterfaceRecord]:
 
     @export
     def ips(self) -> list[IPAddress]:
+        """Return IP addresses as list of :class:`IPAddress`"""
         return list(self._get_record_type("ip"))
 
     @export
     def gateways(self) -> list[IPAddress]:
+        """Return gateways as list of :class:`IPAddress`"""
         return list(self._get_record_type("gateway"))
 
     @export
     def macs(self) -> list[str]:
+        """Return MAC addresses as list of :class:`str`"""
         return list(self._get_record_type("mac"))
 
     @export
     def dns(self) -> list[str]:
+        """Return DNS addresses as list of :class:`str`"""
```

Comment:
```suggestion
        """Return DNS addresses as list of :class:`str`."""
```
Do the `str` and `IPAddress` references here work as expected, or do anything at all?